src/main/webapp/front-end/src/reducer.js

```diff
@@ -28,7 +32,11 @@ function cleanState() {
         markingImageSeen: Map({error: null, loading: false}),
         currentImage: Map({image: null, loadConfigs: null, markedSeen: false, error: null, loading: false}),
         savingLabels: Map({error: null, loading: false}),
-        currentLabels: Map({labels: List.of(), error: null, loading: false})
+        currentLabels: Map({labels: List.of(), error: null, loading: false}),
+        signupEmail: Map({ email: '' }),
+        signupPassword: Map({ password: '' }),
+        loginEmail: Map({ email: '' }),
+        loginPassword: Map({ password: '' })
```

Comment:
what is the best thinking on setting up state?  is it good to have flat properties like this?  more nested?  just interesting..